frontend/packages/kubevirt-plugin/src/components/create-vm/forms/create-vm-form.tsx

```diff
@@ -53,9 +53,9 @@ import {
   URLSource,
 } from '../../vm-templates/vm-template-source';
 import { BootSourceState } from './boot-source-form-reducer';
-import { FORM_ACTION_TYPE, FormAction, FormState } from './create-vm-form-reducer';
-
+import { FormAction, FormState, FORM_ACTION_TYPE } from './create-vm-form-reducer';
 import './create-vm-form.scss';
```

Comment:
- scss imports are last
- if possible (e.g. linting pass) have an empty line to separate from regular imports